modules/parquet/src/parquet-wasm-loader.ts

```diff
@@ -0,0 +1,36 @@
+import type {Loader, LoaderOptions} from '@loaders.gl/loader-utils';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+export type ParquetLoaderOptions = LoaderOptions & {
+  parquet?: {
+    type?: 'arrow-table';
+    wasmUrl?: string;
+  };
+};
+
+const DEFAULT_PARQUET_LOADER_OPTIONS: ParquetLoaderOptions = {
+  parquet: {
+    type: 'arrow-table',
+    wasmUrl: 'https://unpkg.com/parquet-wasm@0.3.1/esm2/arrow1_bg.wasm'
+  }
+};
+
+/** ParquetJS table loader */
+export const ParquetWasmLoader = {
+  name: 'Apache Parquet',
+  id: 'parquet',
+  module: 'parquet',
+  version: VERSION,
+  worker: true,
```

Comment:
Should turn this to `false` for now. Probably related to the failing CI test here: https://github.com/visgl/loaders.gl/runs/6180796447?check_suite_focus=true#step:6:5581